libs/journeys/ui/src/components/StepFooter/FooterButtonList/ReactionButton/ReactionButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement, useState } from 'react'
+import ThumbsUp from '@core/shared/ui/icons/ThumbsUp'
+import ThumbsDown from '@core/shared/ui/icons/ThumbsDown'
+import { StyledFooterButton } from '../StyledFooterButton'
+
+type ReactionVariant = 'thumbsup' | 'thumbsdown'
+
+interface ReactionButtonProps {
+  variant: ReactionVariant
+}
```

Comment:
Any reason we can't just do this?

```suggestion
interface ReactionButtonProps {
  variant: 'thumbsup' | 'thumbsdown'
}
```